wellregistry/registry/admin/monitoring_location.py

```diff
@@ -15,13 +15,164 @@
 from .fetch_from_nwis import FetchFromNwisView
 
 USGS_AGENCY_CD = 'USGS'
-
+tool_tips = [{
```

Comment:
Nit: tooltip is one word so this should be "tooltips" without underscore

Reply:
fixed